packages/@aws-cdk/region-info/build-tools/aws-entities.ts

```diff
@@ -4,32 +4,33 @@
  * Not in the list ==> no built-in data for that region.
  */
 export const AWS_REGIONS = [
-  'us-east-2',
-  'us-east-1',
-  'us-west-1',
-  'us-west-2',
-  'us-gov-east-1',
-  'us-gov-west-1',
-  'us-iso-east-1',
-  'us-isob-east-1',
   'af-south-1',
   'ap-east-1',
-  'ap-south-1',
+  'ap-northeast-1',
   'ap-northeast-2',
+  'ap-northeast-3',
+  'ap-south-1',
   'ap-southeast-1',
   'ap-southeast-2',
-  'ap-northeast-1',
   'ca-central-1',
   'cn-north-1',
   'cn-northwest-1',
   'eu-central-1',
+  'eu-north-1',
+  'eu-south-1',
   'eu-west-1',
   'eu-west-2',
   'eu-west-3',
-  'eu-north-1',
-  'eu-south-1',
   'me-south-1',
   'sa-east-1',
+  'us-east-1',
+  'us-east-2',
+  'us-gov-east-1',
+  'us-gov-west-1',
+  'us-iso-east-1',
+  'us-isob-east-1',
+  'us-west-1',
+  'us-west-2',
 ].sort();
```

Comment:
Added the missing region, and also re-ordered entries in alphanumeric order, so it's easier to browse visually (especially scanning for a missing region, etc...).